app/controllers/api/v1/plot_plants_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::PlotPlantsController < ApplicationController
+  def index
+    plot_plant = PlotPlant.find_plot_plants(params[:plot_id])
```

Comment:
Seems like a more straightforward way to do this would be to find the Plot in the database and then just call `plot.plants`. Would be good for an easy refactor.